corehq/apps/domain/forms.py

```diff
@@ -1979,20 +1979,21 @@ def save(self, commit=True):
     def send_prepayment_invoice(self, date_start, date_end):
         label = f"One month of {self.plan_version.plan.name}"
         monthly_fee = self.plan_version.product_rate.monthly_fee
-        quantity = 12  # todo: pass in as kwarg and/or calculate from dates
-        amount = monthly_fee * quantity
+        duration = relativedelta(date_end, date_start)
+        num_months = getattr(duration, 'years', 0) * 12  # todo: support other subscription lengths
```

Comment:
nit `12` also feels like a "magic" number here...if we keep this around we should at least assign it to `ANNUAL_LENGTH = 12` (or something) elsewhere